kornia/feature/laf.py

```diff
@@ -352,10 +352,10 @@ def extract_patches_from_pyramid(img: torch.Tensor,
                 cur_img[i:i + 1],
                 LAF[i:i + 1, scale_mask, :, :],
                 PS)
-            out[i, scale_mask, ..., :] = out[i, scale_mask, ...,:].clone() * 0\
-                + F.grid_sample(cur_img[i:i + 1].expand(grid.size(0),ch, h, w),
-                                                        grid,
-                                                        padding_mode="border")
+            out[i, scale_mask, ..., :] = out[i, scale_mask, ..., :].clone() * 0\
+                + F.grid_sample(cur_img[i:i + 1].expand(grid.size(0), ch, h, w),
```

Comment:
what did we decide about the operators in this kind of line? before or after the new line? Another thing, new lines should be managed with \ or with (). I do not remember.